ggshield/verticals/sca/sca_scan_models.py

```diff
@@ -88,11 +88,13 @@ class SCAScanAllOutput(Base, FromDictMixin):
     found_package_vulns: List[SCALocationVulnerability] = field(default_factory=list)
 
 
-SCAScanAllOutput.SCHEMA = cast(
-    BaseSchema,
-    marshmallow_dataclass.class_schema(SCAScanAllOutput, base_schema=BaseSchema)(),
+SCAScanAllOutputSchema = cast(
+    Type[BaseSchema],
+    marshmallow_dataclass.class_schema(SCAScanAllOutput, base_schema=BaseSchema),
 )
 
+SCAScanAllOutput.SCHEMA = SCAScanAllOutputSchema()
```

Comment:
why is this change needed?

Reply:
To be able to inherit SCAScanAllOutputSchema in schemas.py to add a total_vulns field